.github/workflows/go-test.yaml

```diff
@@ -179,10 +177,8 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: setup golang
-        uses: actions/setup-go@v5
-        with:
-          go-version: "1.20"
+      - name: setup golang version
```

Comment:
`setup golang ` is enough we dont need to specify version

Reply:
updated